app/controllers/candidate_interface/personal_details/languages_controller.rb

```diff
@@ -21,13 +21,21 @@ def create
 
       def edit
         @languages_form = LanguagesForm.build_from_application(current_application)
+
+        @return_to = if redirect_back_to_application_review_page?
+                       application_review_path_and_params
+                     else
+                       { path: candidate_interface_personal_details_complete_path, params: {} }
+                     end
```

Comment:
Maybe we can put this `if` block behind a method in CandidateInterfaceController. Something like:

```
@return_to = return_to_after_edit(default: candidate_interface_personal_details_complete_path)
...

def return_to_after_edit(default:)
  if redirect_back_to_application_review_page?
    { path: candidate_interface_application_review_path, params: { 'return-to' => 'application-review' } }
  else
    { path: default, params: {} }
  end
end
```

(and then we can get rid of `application_review_path_and_params, which doesn't appear to be needed anywhere else)

Reply:
Nice - like that idea 👍